packages/retail-ui-extensions/src/components/Selectable/Selectable.ts

```diff
@@ -0,0 +1,10 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+export interface SelectableProps {
+  onPress: () => void;
+  disabled?: boolean;
```

Comment:
should we use a default value (true)?

Reply:
By default it should be enabled, so we'll use false. But that's an implementation detail on POS (if they omit this variable)